package/models.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+
+class Package(models.Model):
+    name = models.TextField()
+    price = models.IntegerField()
```

Comment:
i think this can use big integer since there will be larger numbers

Reply:
The integer field goes up to 2 billion, which is quite large considering the maximum price right now is just 10 thousand. I feel like it would be a waste of memory. Plus, 2 billion can buy a house! 🏠 Do you have any other thoughts in mind?

Reply:
Right, after reading your reply I see that it is not necessary. Great analogy btw 🙂‍↕️